Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9
+
+WORKDIR /app
+
+COPY bot.py .
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+CMD ["python", "./bot.py"]
```

Comment:
Please add a newline at the end of file. This is indicated by a ![image](https://user-images.githubusercontent.com/30315706/101144349-33364900-363e-11eb-82e1-c1fcb4940964.png) in the PR info.